File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/notifications/NotificationsActivity.java
package com.jdroid.android.sample.ui.notifications;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class NotificationsActivity extends FragmentContainerActivity {

	/**
	 * @see com.jdroid.android.activity.FragmentContainerActivity#getFragmentClass()
	 */
	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return NotificationsFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/notifications/NotificationsActivity.kt
package com.jdroid.android.sample.ui.notifications

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class NotificationsActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return NotificationsFragment::class.java
    }
}
